airbyte-integrations/connectors/source-sentry/unit_tests/integration/test_issues_stream.py

```diff
@@ -31,24 +31,35 @@ def test_read(self, http_mocker: HttpMocker):
         http_mocker.get(
             HttpRequest(
                 url="https://sentry.io/api/0/projects/test%20organization/test%20project/issues/",
-                query_params={"query": "lastSeen:>1900-01-01T00:00:00.0Z"}
+                query_params={"query": "lastSeen:>1900-01-01T00:00:00.000000Z"}
```

Comment:
i've tested this locally and sending in a micro second precision of 6 will work w/ sentry's API